### DISCUSSION TITLE
customizing the BuildConfig in Kafka Connect

### DISCUSSION THREAD

**Discussion by 282ori** [strimzi/strimzi-kafka-operator#6621] [Q&A]
Hi,
I'm trying to use artificats stored in a private registry outside my OpenShift cluster as plugins for Kafka Connect.
The registry has a certificate signed by a private CA and I want my build pod to trust this certificate.
I prefer not to run it with an insecure flag but I don't have any other way to run it as of now.
This can be easily achieved by allowing to customize the BuildConfig within the KafkaConnect CR (and not only its metadata), and I'm sure it would have more use cases.
If there is another way to trust a custom CA though, I'd like to know.

**Comment by scholzj** [comment]
It is not really clear what artifacts or registry are you talking about. But you can also just build the image your self using your own Dockerfile to workaround any limitations caused by private repositories.
